packages/commonwealth/client/scripts/state/api/comments/createReaction.ts

```diff
@@ -87,7 +88,14 @@ const useCreateCommentReactionMutation = ({
 
       return reaction;
     },
-    onError: (error) => checkForSessionKeyRevalidationErrors(error),
+    onError: (error) => {
+      if (error instanceof AxiosError) {
+        if (error.response?.data?.error?.toLowerCase().includes('stake')) {
```

Comment:
is there a more type safe way to check for stake related errors?

Reply:
Yeah thought about that. Do we have a convention yet for sharing constants/errors between BE/FE?

Reply:
we have libs/shared and errors can carry specific codes or other details that are better suited for this

Reply:
Will tackle this in a subsequent PR next time we need to share errors. Spent too much time on fixing the tests today.